packages/unimodules-test-core/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "unimodules-test-core",
+  "version": "0.1.0",
+  "description": "Module providing native testing utilities for testing unimodules",
+  "keywords": [
+    "react-native",
+    "expo",
+    "unimodules",
+    "unimodules-test-core"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/unimodules-test-core"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/unimodules-test-core"
+}
```

Comment:
I believe we need proper file configurations across all filetypes/IDEs. Newline here